packages/components/src/Tooltip/Tooltip.test.tsx

```diff
@@ -31,6 +34,59 @@ test("tooltip should show up on hover", () => {
   act(() => {
     getByTestId(contentID).dispatchEvent(new MouseEvent("mouseenter"));
   });
-  expect(getByText(message)).toBeTruthy();
-  expect(getByText(content)).toBeTruthy();
+  expect(getByText(message)).toBeInTheDocument();
+  expect(getByText(content)).toBeInTheDocument();
+});
+
+test("tooltip should show up on focus", () => {
+  const message = "Tipping the tool on focus";
+  const content = "Focus on me";
+  const contentID = "focus-on-me";
+
+  const { getByTestId, getByText } = render(
+    <Tooltip message={message}>
+      <div data-testid={contentID}>{content}</div>
+    </Tooltip>,
+  );
+
+  act(() => {
+    getByTestId(contentID).dispatchEvent(new Event("focus"));
+  });
```

Comment:
I don't think we should need to use `act` directly and can use `waitFor` instead

Reply:
I'll do you one better. I used the `userEvent` instead and not use `act`anymore